src/libs/OptionsListUtils.js

```diff
@@ -393,6 +397,23 @@ function getLastMessageTextForReport(report) {
             lastMessageTextFromReport = lodashGet(latestVisibleAction, 'message[0].text', '');
         }
     }
+    if (ReportActionUtils.isReportPreviewAction(lastReportAction)) {
```

Comment:
This code addition seems to be a merge miss? @tienifr

Reply:
Kindly do let me know once this is sorted out. Will do a another review again. @tienifr